controller/controller.go

```diff
@@ -569,27 +569,36 @@ func contentInstall(rootDir string, version string,
 		bundles = append(bundles, md.Kernel.Bundle)
 	}
 
+	// We have offline content available
 	if swupd.IsOfflineContent() {
-		msg := utils.Locale.Get("Copying cached content to target media")
-		prg = progress.NewLoop(msg)
-		log.Info(msg)
-
 		if err := utils.ParseOSClearVersion(); err != nil {
 			return prg, err
 		}
 
-		if version == "latest" || version == utils.ClearVersion {
+		// If command line did not explicitly set "latest", use the current
+		// image version to enable Offline content usage
+		// Note: Setting "version: 0" in the YAML will not override offline content
+		if version == "latest" && options.SwupdVersion == "" {
+			version = utils.ClearVersion
+			log.Info("Overriding version from 'latest' to %s to enable offline install", utils.ClearVersion)
+		}
+
+		if version == utils.ClearVersion {
+			msg := utils.Locale.Get("Copying cached content to target media")
+			prg = progress.NewLoop(msg)
```

Comment:
I know we used to do that before, but just pointing out, we will be reporting progress success for offline content copy even if copyOfflineToStatedir operation has an error(I know its a soft error but it gives ). 

This is just for discussion. No action needed

Reply:
Updated to show a failure, but continue.

TUI:
<img width="825" alt="cache_fail_tui" src="https://user-images.githubusercontent.com/12535259/82358711-c027f680-99bb-11ea-9d8b-d0e39f57ce5b.png">

GUI:
![cache_fail_gui](https://user-images.githubusercontent.com/12535259/82358720-c3bb7d80-99bb-11ea-9eaa-b9bb403f3990.png)